### DISCUSSION TITLE
Support for EBS Direct API?

### DISCUSSION THREAD

**Discussion by joshtriplett** [awslabs/aws-sdk-rust#58] [General]
I'd love to have support for the [EBS Direct API](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-accessing-snapshot.html), for manually constructing a snapshot by uploading blocks.

I currently have a snapshot-management tool based on rusoto that I'd love to port to the new SDK.

**Comment by rcoh** [comment]
I believe this should be included with #66 (EBS support), I looked into it, I don't think that the direct APIs are separate from the other EBS APIs (AFAICT)

**Reply by joshtriplett** [reply]
Yes, it should. #66 hasn't been filed when I posted this. 🙂
